opm/output/eclipse/Summary.cpp

```diff
@@ -432,6 +458,93 @@ inline double well_keywords( const smspec_node_type* node,
     }
 }
 
+inline double sum( const std::vector< const data::Well* >& wells, rt phase ) {
+    double res = 0;
+
+    for( const auto* well : wells )
+        res += well->rates.get( phase, 0 );
+
+    return res;
+}
+
+
+inline double sum_rate( const std::vector< const data::Well* >& wells,
+                   rt phase,
+                   const double* conversion_table ) {
+
+    switch( phase ) {
+        case rt::wat:
+        case rt::oil: return convert( sum( wells, phase ),
+                                      dim::liquid_surface_rate,
+                                      conversion_table );
+
+        case rt::gas: return convert( sum( wells, phase ),
+                                      dim::gas_surface_rate,
+                                      conversion_table );
+        default: return sum( wells, phase );
+    }
+}
+
+inline double sum_vol( const std::vector< const data::Well* >& wells,
+                   rt phase,
+                   const double* conversion_table ) {
+
+    switch( phase ) {
+        case rt::wat:
+        case rt::oil: return convert( sum( wells, phase ),
+                                      dim::liquid_surface_volume,
+                                      conversion_table );
+
+        case rt::gas: return convert( sum( wells, phase ),
+                                      dim::gas_surface_volume,
+                                      conversion_table );
+        default: return sum( wells, phase );
```

Comment:
Ideally nothing, and it could be a throw. The fall-back here is just not applying any unit conversions. Preferences?